Dangerfile

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+def release_branch?
+  danger.github.branch_for_base.start_with?('release/') || danger.github.branch_for_base.start_with?('hotfix/')
+end
+
+def main_branch?
+  danger.github.branch_for_base == 'trunk'
+end
+
+def wip_feature?
+  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
+  has_wip_title = github.pr_title.include?('WIP')
+
+  has_wip_label || has_wip_title
+end
+
+return if github.pr_labels.include?('Releases')
+
+github.dismiss_out_of_range_messages
+
+manifest_pr_checker.check_gemfile_lock_updated
+
+labels_checker.check(
+  required_labels: [//],
+  required_labels_error: 'PR requires at least one label.'
+)
```

Comment:
Did you consider adding support for "do not merge" here, even though it wasn't there before?

Reply:
The plugin has a parameter for a "do not merge label", and the default is `Do Not Merge` (case insensitive) which matches several of our projects. So if you don't specify anything, it's a check you'll get -- some projects (e.g. [Woo iOS](https://github.com/woocommerce/woocommerce-ios/pull/10654/files#diff-c429cc891087188b07403da9305b15810b0b14e3ad75cef11bdf09580c4734ddR25)) use a different label, and in those cases we need to specify the parameter.

The [Peril configuration](https://github.com/Automattic/peril-settings/blob/2fb61517b92205323c96c30b90bb4307ef6b8f13/org/pr/label.ts#L13)  actually does something like that as well.

Reply:
Gotcha.

🤔 Shame on me for forgetting about this. But it does reveal a somewhat hidden default behavior. All good in the context of a tool for internal usage.

Reply:
> But it does reveal a somewhat hidden default behavior. 

Very true -- I also noticed that some of these defaults might hide too much what is going to happen. It's worth it taking a second look at them in any case.

Reply:
> But it does reveal a somewhat hidden default behavior. All good in the context of a tool for internal usage.

@mokagio We really do have the same mind 😂 

https://github.tumblr.net/TumblrMobile/android2/pull/21109#discussion_r472272
https://github.tumblr.net/TumblrMobile/android2/pull/21109#discussion_r472275

Reply:
> @mokagio We really do have the same mind 😂

Ha, yes 😂 

By the way, I forgot to mention there's [an issue](https://github.com/orgs/Automattic/projects/775/views/1?pane=issue&itemId=42524204) about the default values that I created when @AliSoftware pointed out the same.

Reply:
@AliSoftware @iangmaia 😄 👌